SzczepionkaWEB/src/app/component/map/map.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {FormBuilder} from '@angular/forms';
+import {RegisterService} from '../../service/register/register.service';
+
+@Component({
+  selector: 'app-map',
+  templateUrl: './map.component.html',
+  styleUrls: ['./map.component.css']
+})
+export class MapComponent implements OnInit {
+
+  constructor(private registerService: RegisterService) { }
+
+  registerForm: any;
+
+  ngOnInit(): void {
+    this.registerForm = this.registerService.getRegisterForm();
+    console.log(this.registerForm);
```

Comment:
I suppose that this line can be removed :D

Reply:
Done with extra clean code things.